### DISCUSSION TITLE
Do you like separate sites window?

### DISCUSSION THREAD

**Discussion by sammyaxe** [beyondcode/herd-community#885] [Polls]
Honestly, I can't get over this new update which separated sites UI out from the main window.
Just wondering what others think of it.

**Comment by Abhishek0116** [comment]
I do not have separate sites on Mac, but most of my communities use Windows, and when I suggested Herd and also tried on my Windows Virtual Machine then YES I found a separate window for sites. I think integrating with the same window was a good option if there is no strong reason to separate it.
